### DISCUSSION TITLE
How to propagate event from deepest actor to topmost actor?

### DISCUSSION THREAD

**Discussion by Alexandreee100** [statelyai/xstate#3754] [Q&A]
Hello, i am new in XState, may you help me, please? I have actor tree, can i propagate event from the deepest actor to the topmost without explicit handling this event on each level between these two actors.

Let say, now i use sendParent action on deepest actor and to pass its event to topmost actor, i should handle this event sent by sendParent on every level between this two actors explicitly.

I want to avoid explicit handling on every level and forward up to this event until top most actor does not catch it

**Comment by davidkpiano** [comment]
Unfortunately this is currently difficult to do, but we are probably introducing the concept of a "receptionist" (actor model) to help with this in v5: https://github.com/statelyai/rfcs/pull/5
